src/modules/list.marko

```diff
@@ -0,0 +1,314 @@
+<div id="list">
+    <section-header metadata=input.metadata name=input.name/>
+    <p>A list component is used to display certain types of data in a structured format. It can be used to display a list of items, a list of links, or a list of buttons.</p>
+
+
+    <h3 id="list-default">Default List</h3>
+   <div class="demo">
+        <div class="demo__inner">
+            <div class="list">
+                <div class="list__item">
+                    <span class="list__leading">
```

Comment:
Shouldn't `leading` be a descendent from `item`, not `list` ?

Grandchildren are a bit more difficult to handle. 

Maybe this could be structured like this?:

```
.list
  .list-item
    .list-item__leading
    .list-item__body
    .list-item__trailing
```

Reply:
Structure had changed, we use `ul` and `li` now.

Reply:
We no longer have `list__item` anymore. Its not needed
We can do `list__item-leading` and such if its more explicit.